lldb/packages/Python/lldbsuite/test/builders/builder.py

```diff
@@ -40,7 +40,9 @@ def getMake(self, test_subdir, test_name):
         """Returns the invocation for GNU make.
         The first argument is a tuple of the relative path to the testcase
         and its filename stem."""
-        if platform.system() == "FreeBSD" or platform.system() == "NetBSD":
+        if configuration.make_path is not None:
+            make = configuration.make_path
+        elif platform.system() == "FreeBSD" or platform.system() == "NetBSD":
             make = "gmake"
         else:
             make = "make"
```

Comment:
I think it'd be nicer to move this code to dotest.py, as that's where we deal with locating the compiler, and all the other tools.

Reply:
Updated MR